src/app/login-register/login-register.component.ts

```diff
@@ -2,20 +2,112 @@ import { Component, OnInit } from '@angular/core';
 import {MatIconModule} from '@angular/material/icon';
 import {MatFormFieldModule} from '@angular/material/form-field';
 import {MatCheckboxModule} from '@angular/material/checkbox';
+import {FormControl, Validators, ReactiveFormsModule} from '@angular/forms';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7552947)
```suggestion
import {FormControl, Validators, ReactiveFormsModule} from "@angular/forms";
```